eZ/Publish/Core/Helper/PreviewLocationProvider.php

```diff
@@ -50,14 +52,37 @@ public function __construct(
      *
      * If the content doesn't have a location nor a location draft, null is returned.
      *
+     * @deprecated Since 7.5.4, rather use {@see loadMainLocationByContent}.
```

Comment:
FYI: inline `@see` does not work with direct navigation in source code. 
What usually do is:
```suggestion
     * @deprecated Since 7.5.4, rather use loadMainLocationByContent.
     * @see loadMainLocationByContent
```
But it works in PhpStorm doc tooltip and gives better DX there, so I'm not insisting :)